remoteworkitem/gorm_transaction.go

```diff
@@ -6,7 +6,7 @@ import (
 
 // NewGormTransactionSupport constructs a new instance of GormTransactionSupport
 func NewGormTransactionSupport(db *gorm.DB) *GormTransactionSupport {
-	return &GormTransactionSupport{db: db}
+	return &GormTransactionSupport{db: db, tx: db}
```

Comment:
I think the `tx` should be explicitly initialized with nil. Later during the beginning of transaction, it should get a `gorm.DB` address. During commit and rollback we are already setting back to nil.

Also we should enforce CRUD is taking place within a transaction. I have added another issue for this: https://github.com/almighty/almighty-core/issues/247